h/static/scripts/annotator/guest.coffee

```diff
@@ -298,7 +323,8 @@ module.exports = class Guest extends Annotator
     if immediate
       # Create an annotation
       @onAdderClick event
-    else
+    # We don't show the adder on mobile since it is not correctly placed and competes with native popups.
+    else if not ('ontouchstart' in window)
```

Comment:
This should be `true` on any device that has a touch screen. For example, if I open this in Windows 8 in Chrome on my laptop I won't get an adder.

I think you should make two separate functions that you bind independently to each type of event. This method would no longer show the adder. Instead, both event handlers would call this method to extract the selection and only one of them would then show the adder.